base/src/main/kotlin/io/spine/string/Strings.kt

```diff
@@ -234,6 +234,21 @@ public fun String.ensurePrefix(prefix: String): String {
  */
 public inline  fun <reified T> simply(): String = T::class.simpleName!!
 
+/**
+ * Returns the simple class name of the given [instance].
+ */
+public fun simpleNameOf(instance: Any): String = instance::class.simpleName!!
```

Comment:
Please have a look at this [implementation in a separate PR](https://github.com/SpineEventEngine/base/pull/878/files#diff-59f9a5aad52d5617e25ec7aa557dcd15cc4a24fd09906742cd42a0e10f693f7a) and associated test.